exporters/trace/src/main/java/com/google/cloud/opentelemetry/trace/TraceConfiguration.java

```diff
@@ -0,0 +1,169 @@
+package com.google.cloud.opentelemetry.trace;
+
+import com.google.auth.Credentials;
+import com.google.auto.value.AutoValue;
+import com.google.cloud.ServiceOptions;
+import com.google.cloud.trace.v2.stub.TraceServiceStub;
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.Preconditions;
+import com.google.common.base.Strings;
+import com.google.devtools.cloudtrace.v2.AttributeValue;
+import java.util.Collections;
+import java.time.Duration;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import javax.annotation.Nullable;
+import javax.annotation.concurrent.Immutable;
+
+/**
+ * Configurations for {@link TraceConfiguration}.
+ *
+ */
+@AutoValue
+@Immutable
+public abstract class TraceConfiguration {
+
+    private static final String DEFAULT_PROJECT_ID = Strings.nullToEmpty(ServiceOptions.getDefaultProjectId());
```

Comment:
The format here looks a bit weird and so do some other places. Would you please apply the google-java-format checker or (if you have) check why it's not working?

Reply:
I applied it. It looks a bit weirder than before in my opinion, but it's following a consistent standard so I suppose its better.